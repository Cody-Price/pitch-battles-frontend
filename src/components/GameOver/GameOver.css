.game-over {
  background: url("../Staff/staff/staff-frame.png");
  background-size: cover;
  background-repeat: no-repeat;
  image-rendering: pixelated;
  width: 345px;
  height: 330px;
  margin: auto;
  left: 0;
  right: 0;
  position: absolute;
  overflow: hidden;
}

.game-over-text {
  text-align: center;
  font-family: "chronotype";
  font-size: 3rem;
  animation: slide-in-victory 0.1s linear;
  position: absolute;
  left: 0;
  right: 0;
}

.victory-screen p {
  text-align: center;
  font-family: "chronotype";
  width: 313px;
  padding: 1rem;
  font-size: 2rem;
  animation: slide-in-victory 0.1s linear;
  position: absolute;
  top: 50px;
  left: 0;
  right: 0;
}

.victory-screen .level-time {
  top: 150px;
}

.victory-screen .total-time {
  position: absolute;
  top: 200px;
}

.exit-message {
  margin-top: 150px;
  font-family: "chronotype";
  font-size: 1.4rem;
}

@keyframes slide-in-victory {
  0% {
    left: -300px;
  }
  100% {
    left: 0;
    right: 0;
  }
}
