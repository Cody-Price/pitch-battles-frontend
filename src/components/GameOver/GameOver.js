import React, { Component } from "react";
import "./GameOver.css";

class GameOver extends Component {
  render() {
    return (
      <section className="game-over">
        <h4 className="game-over-text">GAME OVER</h4>
      </section>
    );
  }
}

export default GameOver;
