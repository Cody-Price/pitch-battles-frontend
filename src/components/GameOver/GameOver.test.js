import React from "react";
import ReactDOM from "react-dom";
import GameOver from "./GameOver";

it("is a test", () => {});
