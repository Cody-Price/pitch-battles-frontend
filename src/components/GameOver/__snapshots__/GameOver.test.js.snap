// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameOver should match the snapshot 1`] = `
<section
  className="game-over"
>
  <h4
    className="game-over-text"
  >
    GAME OVER
  </h4>
  <p
    className="exit-message"
  >
    click here to exit...
  </p>
</section>
`;
